src/components/table/table.comp.tsx

```diff
@@ -71,6 +72,12 @@ export const Table = ({ items, fields, callbacks, customActions, customLabels }:
                   }
                   <td>
                     <div className="actions-wrapper">
+                      {
+                        callbacks.details &&
```

Comment:
Well here it's because the button redirects you to the details page of the item. While `put` and `delete` actually act on the item, the detail button takes you out of the page. 

So I thought using `get` would be confusing. But I guess so is `details`. Either `get`to be consistent with the method used behind or `goToDetails` to be explicit ?